docs/modules/install/pages/index.adoc

```diff
@@ -0,0 +1,63 @@
+= Getting started with Decidim Bulletin Board
+:source-highlighter: highlightjs
+
+== What is and what isn't Decidim Bulletin Board?
+
+Decidim Bulletin Board is an independent application designed to provide services to Decidim instances to allow them to celebrate secure end-to-end auditable votings. But it can't be used as a standalone application, since it doesn't offer a user interface organization administrators to define elections nor for voters to cast their votes.
+
+If you want to start your own installation of Decidim Bulletin Board, you need to clone this repo. Keep reading to find out how to install Decidim Bulletin Board.
+
+== Clone the Decidim Bulletin Board repository
+
+The Decidim Bulletin Board repository includes the main Rails application, but also the Ruby client and the JS client to be used inside Decidim apps, and additional projects for the different voting schemes implemented.
+
+So, to be able to run the application, first you have to clone the repository:
+
+[source,console]
+----
+git clone git@github.com:decidim/decidim-bulletin-board.git
+cd decidim-bulletin-board
+----
+
+And then you have to enter to the Rails application directory:
+
+[source,console]
+----
+cd decidim-bulletin-board-app
+bundle install
+yarn install
+----
+
+== Initializing your app for local development
+
+You should now setup your database:
+
+[source,console]
+----
+bin/rails db:create db:migrate db:seed
+----
+
+This will also create some default data so you can start testing the app.
+
+This data won't be created in production environments, if you still want to do it, run: `$ SEED=true rails db:setup`.
+
+You can now start your server!
+
+[source,console]
+----
+bin/rails s
+----
+
+Visit http://localhost:8000 to see your app running.
```

Comment:
Maybe we should add the link to the sandbox and graphiql? You barely see anything on localhost:8000.

Reply:
@slickepinne since #98 was merged there's an index page with links to the sections you mention! ✨